cmd/soroban-rpc/internal/test/get_version_info_test.go

```diff
@@ -0,0 +1,68 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/config"
+	"os/exec"
+	"testing"
+
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/jhttp"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/methods"
+)
+
+func TestGetVersionInfoSucceeds(t *testing.T) {
+	test := NewTest(t, nil)
+
+	populateVersionInfo(test)
+	defer resetVersionInfo(test)
+
+	ch := jhttp.NewChannel(test.sorobanRPCURL(), nil)
+	client := jrpc2.NewClient(ch, nil)
+	request := methods.GetVersionInfoRequest{}
+
+	var result methods.GetVersionInfoResponse
+	err := client.CallResult(context.Background(), "getVersionInfo", request, &result)
+	assert.NoError(t, err)
+
+	assert.NotEmpty(t, result.Version)
+	assert.NotEmpty(t, result.BuildTimestamp)
+	assert.NotEmpty(t, result.CommitHash)
+	assert.NotEmpty(t, result.CaptiveCoreVersion)
+	assert.NotEmpty(t, result.ProtocolVersion)
```

Comment:
the protocol version is available in `test` . we should assert that `result.ProtocolVersion` matches the expected protocol version